bool create_scene
bool clean_scene
geometry_msgs/PoseStamped object_pose_world
string object_mesh_path
---
bool success